third-party/realdds/src/dds-device-server.cpp

```diff
@@ -120,11 +141,27 @@ void dds_device_server::init( std::vector< std::shared_ptr< dds_stream_server >
         }
 
         _notification_server->run();
+
+        // Create a control reader and set callback
+        auto topic = topics::flexible_msg::create_topic( _subscriber->get_participant(), _topic_root + "/control" );
+        _control_reader = std::make_shared< dds_topic_reader >( topic, _subscriber );
+
+        dds_topic_reader::qos rqos( RELIABLE_RELIABILITY_QOS );
+        rqos.history().depth = 10; // default is 1
+        // Does not allocate for every sample but still gives flexibility. See:
+        //     https://github.com/eProsima/Fast-DDS/discussions/2707
+        // (default is PREALLOCATED_MEMORY_MODE)
+        rqos.endpoint().history_memory_policy = eprosima::fastrtps::rtps::PREALLOCATED_WITH_REALLOC_MEMORY_MODE;
```

Comment:
This is our default. Any special need to mention it again here?

Reply:
No, I will remove.
The rs-dds-sniffer does not receive the notification messages and this are leftovers from some things I tried to fix it

Reply:
Removed